README.md

```diff
@@ -511,7 +532,25 @@ block-beta
 ```bash
 protoflow execute WriteFile path=/tmp/file.txt
 ```
+#### [`WriteSocket`]
 
+A block that writes proto messages to TCP port.
+
+```mermaid
+block-beta
+    columns 4
+    Source space:2 WriteSocket
+    Source-- "input" -->WriteSocket
+
+    classDef block height:48px,padding:8px;
+    classDef hidden visibility:none;
+    class WriteSocket block
+    class Source hidden
+```
+
+```bash
+protoflow execute WriteSocket host="127.0.0.1" port=7077 buffer_size=1024
```

Comment:
The quotes around the host value are unnecessary here

Reply:
fixed